### PR TITLE
Created Contact Us Form

### PR DESCRIPTION
**Pull request by anchal-gupta**
Added contact us form and contact us page

### CODE REVIEW

**Comment by anchal-gupta** [comment]
@PrajaktaSathe Can you please review these updates ?

File: index.html [hunk 1]
```diff
@@ -129,18 +129,11 @@ <h4 class="section-title" style="font-weight: 500;">About Us</h4>
         <div>Lorem ipsum dolor sit amet consectetur adipisicing elit. Illo sint nulla rerum dolorem! Dolore non amet ullam blanditiis alias eaque soluta. Doloribus id praesentium beatae. Perspiciatis corrupti quibusdam laudantium consectetur.</div>
     </div>
 
-    <!-- Contact Us Section -->
```
Comment (PrajaktaSathe) [inline index.html:132]: Please don't remove this section.

**Comment by PrajaktaSathe** [comment]
Please resolve the conflicts. The conflicting file is index.html.

File: index.html (hunk 1 above)
Comment (anchal-gupta) [reply index.html:132]: I have created a new "Contact Us" Page and have attached a contact us form to that page. Do you still want this section to be in home page instead ?

Comment (PrajaktaSathe) [reply index.html:132]: I don't want to change the current main page outlook. I want it to be as it is for now. Sorry. I hope that's ok.

**Pull request closed without merging**